app/controllers/submissions_controller.rb

```diff
@@ -126,8 +128,8 @@ def file_manager
     set_filebrowser_vars(user_group, @assignment)
   end
 
-  def populate_file_manager
-    @assignment = Assignment.find(params[:assignment_id])
+  def populate_file_manager_react
```

Comment:
Assignment Branch Condition size for populate_file_manager_react is too high. [38/15]